### PR TITLE
Use latest react

### PR DESCRIPTION
**Pull request by stegben**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +59,7 @@
     "react-faux-dom": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0"
+    "react": "^0.14.0 || ^15.0.0-0 || 16.0.0"
```
Comment (tw0517tw) [inline package.json]: `^16.0.0` ?

Comment (chentsulin) [inline package.json]: use `^16.0.0-0` let it ca support alpha, beta versions

Comment (stegben) [inline package.json]: done

File: package.json [hunk 2]
```diff
@@ -58,7 +59,7 @@
     "react-faux-dom": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0"
+    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0"
```
Comment (chentsulin) [inline package.json]: ^16.0.0-0

**Pull request merged**